### DISCUSSION TITLE
[archived] "Argo CLI Walk Through" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#9554] [General]
# argo-workflows/walk-through/argo-cli/

https://argoproj.github.io/argo-workflows/walk-through/argo-cli/

**Comment by alramdein** [comment]
how to install it? I think you guys better put it in here or atleast point to the page about how to install it
https://github.com/argoproj/argo-workflows/releases/tag/v3.3.9

**Reply by agilgur5** [reply]
A link to the releases was added to this page in #11751.

If you'd like to make improvements to the docs, PRs always welcome!

**Comment by sarabala1979** [comment]
I think you need to start from quick start

https://argoproj.github.io/argo-workflows/quick-start/
